README.md

```diff
@@ -743,26 +743,45 @@ test('When asked for an existing order, Then should retrieve it and receive 200
 <br/>
 
 <details><summary>✏ <b>Code Examples</b></summary>
-//docker-compose file
 
+Metadata:
+
+```javascript
+module.exports = async () => {
+  console.time('global-setup');
+  ...
+  await npmCommandAsPromise(['db:seed']);
```

Comment:
Add comment to clarify this - "This will add a countries list (not related to the test records, just global metadata)"